ddp-workspace/projects/ddp-sdk/src/lib/components/activityForm/picklist/autocompleteActivityPicklistQuestion.component.ts

```diff
@@ -150,6 +158,11 @@ export class AutocompleteActivityPicklistQuestion extends BaseActivityPicklistQu
         return options.filter(option => option.optionLabel.toLowerCase().includes(filterValue));
     }
 
+    private get shouldBeSorted(): boolean {
+        const doNotSort = this.studyGuid === 'cmi-pancan' && this.DO_NOT_SORT_LIST_STABLE_IDS.includes(this.block.stableId);
```

Comment:
what do you think if instead of specifying exception study id and the list of exception picklists ids we can do one this things
1. Add new model property to question json, for example noSorting: boolean and specify on backend this field as true for prequlifier cancers list in config. Then check this field on frontend
OR if changes on backend would be hard to implement
2. Add new property in ConfigurationService (for example picklistsWithNoSorting: string[]) for list of ids for every study and then check on this sdk component if study has such configuration and do not sort picklists from that list

Reply:
As I wrote in [the PR description](https://github.com/broadinstitute/ddp-angular/pull/813#issue-713306386):
  > Implemented a temporary solution on client-side. (because we're running out of time)

For now we are using Autocomplete component in Pancan study only. Thus having list of `picklistsWithNoSorting` ids  either in config or in the component itself seem pretty the same, being a temporary decision.

More optimal way is your suggestion#1: get sorting field for `Autocomplete picklist` from back-end as a separate field. 
https://github.com/broadinstitute/ddp-angular/pull/813#:~:text=I%20believe%2C%20generally%20(in%20future)%20we%20should%20have%20several%20sorting%20options%20on%20client-side%20and%20on%20back-end%20set%20special%20flag/field%20for%20ActivityPicklistQuestion%2C%20like%20sort%3A%20NONE%20/%20ALPHABET%20/%20ANOTHER%20SORTING.

Reply:
you're right that it's much better having the backend configuration for that, but as far as I understand there is no task in backlog for moving the sorting config on a backend, which means it would be on FE for uncertain time. 
I'd recommend to keep study specific configs in ConfigurationService (app.module.ts) even for temporal solutions inctead of keeping it right in sdk component. It will help to keep sdk component clean and simple. Moreover each sdk component has access to ConfigurationService inctance and can easily extract any study specific configuration

Reply:
RIght. Was hoping for solution that kept the SDK code without dependencies to Pan Can.
The piece that eventually would make sense to make part of the platform and include in our API is that our study configuration can specify, for each picklist question, a sort policy. We don't have time to do that, so what we need to do in client is to hard-code the knowledge of which question has picklist options that need to be sorted and which questions do not. 
I think we can do better than hard-code the stable ids into the component.
As you recall, to add the SortPolicy in the first place I went to great lengths to make sure it was something that could be specified outside of the component source code. I hope that we can do something similar for this ticket.
It ought to be possible using a combination configuration service and (maybe, but hopefully not) Angular service injection.
IMPORTANT! If this looks too hard to finish by today, this is OK. Will just go with the hard-coded solution and fix after launch. We have ALMOST run out of time here.

Reply:
Fixed.